fdi/findings_data_import.py

```diff
@@ -196,9 +198,12 @@ def import_data(
                     finding.pop(field, None)
 
             # Get RVA ID in format DDDD(.D+) from the end of the "RVA ID" field.
```

Comment:
I think this comment needs to be updated, since the regex has changed.

Reply:
Good catch, fixed in 370bfae